examples/specs/layer_candlestick.vl.json

```diff
@@ -1,8 +1,13 @@
 {
   "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
   "width": 320,
-  "description":"A candlestick chart inspired from Protovis(http://mbostock.github.io/protovis/ex/candlestick.html)",
+  "description": "A candlestick chart inspired from Protovis(http://mbostock.github.io/protovis/ex/candlestick.html)",
   "data": {
+    "format": {
+      "parse": {
+        "date": "date:'%Y-%m-%d'"
```

Comment:
Browsers should parse this automatically. Why do you need to add the format?

Reply:
Its parsing as utc

Reply:
Vega & Vega-Lite simply uses JS parsing.  The behavior is consistent with the screenshot you send.  But time scale is default local time.  So if you want to visualize local time, need to parse correctly.

Reply:
It's a bit weird that we `utc:FORMAT`, `utc`, and `date` parse as UTC but `date:FORMAT` parses as local, though.